src/apps/frontend/components/otp/index.tsx

```diff
@@ -8,7 +8,7 @@ import Flex from '../flex/flex.component';
 import OTPInput from './otp-input';
 
 interface OTPProps {
-  error: string;
+  error: string | false | undefined;
```

Comment:
Why would this be false?

This should be `error?: string` or `error: Nullable<string>`